about.html

```diff
@@ -0,0 +1,149 @@
+              </p>
+      <p class="about-description desk-version">
+        Not only do we offer high-quality, career-focused study programmes that prepare you well for the working world, but you can be assured that they are up to the highest educational standards. 
+        At IU you get a high-quality degree from Germany’s largest university. But what makes us even prouder, is that 96% of our students think so too.
+        Our practice-oriented study programs prepare you for your future career. This is why 94% of IU graduates find a job within 3 months after graduation.
+      </p>
+      <div class="contact">
+        <p class="contact-p">
+          Please contact us per Email for any further questions about
+          the Get to know about PCC 2024! 
+          <br> 
+          <a href="#" class="contact-email" >mohammedjamal999999@gmail.com</a>
+        </p>
+      </div>  
+    </div>
+  </section>
+  
+  <section class="iu-logo">
+    <h3>Get to know about IU for better opportunities</h3>
+    <div class="mid">
+      <hr class="line"/>
+    </div>
+    <p class="mobile-version">
+      At IU, it has always been our mission to help as many people as possible to take control of their career and future.
+    </p>
+    <p class="desk-version">
+      At IU, it has always been our mission to help as many people as possible to take control of their career and future.
+      Would you like to get an overview of our study programmes? Would you like to find out more about IU International University of Applied Sciences and about studying online or on campus with us? Simply register for one of our information events.
+    </p>
+    <img src="./images/iu-small-logo.png" alt="IU logo" class="iu-logo-img"/>
+    <br class="logo-break"/>
+    <br/>
+    <hr class="full-line"/>
+  </section>
+  <section class="PastEvents">
+    <h3>See the past PCC events</h3>
+    <div class="mid">
+      <hr class="line"/>
+    </div>
+    <p class="EventsDescription">Take a look at the two past events "PCC 5, PCC 6" which took place in Alexandria Egypt.</p>
+    <div class="events-container">
+        <div class="Event-alg event-content">
+            <h3>2023</h3>
+            <p>PCC 6, Konouz Agamy</p>
+        </div>
+        <div class="Event-mor event-content">
+            <h3>2022</h3>
+            <p>Bibleotheca Alexandria, Shatbi</p>
+        </div>
+    </div>
+</section>
+
+<section class="partners-conatainer">
+  <h3>Partners</h3>
+  <div class="mid">
+    <hr class="line"/>
+  </div>
+  <div class="Partners-logos">
+    <i class="fab fa-aws fa-4x"></i>
+    <i class="fab fa-google fa-4x"></i>
+    <i class="fab fa-microsoft fa-4x"></i>
+    <i class="fab fa-reddit fa-4x"></i>
+    <i class="fab fa-facebook fa-4x"></i>
+  </div>
+</section>
+
+<footer class="about-footer">
+  <img src="./images/iu-small-logo.png" alt="" />
+  <div class="footer-text">
+    <p> 2021 Get to know about the IU <br> 
+   Some Rights Reserved</p>
+  </div>
+</footer>
```

Comment:
Excellent job on the footer section! 👍 I have some suggestions to **make it closely resemble the [provided design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and further enhance its appearance**:

- [ ] Could you please add the correct `footer` to your home page in the mobile version? Currently, **it is not visible, and including it will ensure consistency across all pages.** 😊

- [ ] Please double-check the `size` of your logo in the `footer` of the home page in the **desktop version**. It appears slightly larger compared to the `footer` on the About page. Adjusting the size will help **maintain a consistent and visually pleasing design.** 💪

- [ ] There seems to be **a peculiar blank space below your `footer`** on the About page. It would be great if you **could remove this space to match the [given design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) accurately.** 🚀

  These suggestions will contribute to creating a polished and cohesive look for your footer section.  💪

   ### Home Page - Desktop
   ![image](https://github.com/mohamedgamaleldin999999/Capstone-project-module-one/assets/107952724/beef594a-cba3-4096-8f20-d0464b634990)


   ### About Page- Desktop
  ![image](https://github.com/mohamedgamaleldin999999/Capstone-project-module-one/assets/107952724/4eee18c3-d73c-4eab-9e96-53820194f0fc)

Reply:
@taldr27  thanks your comments are amazing. btw, the mobile footer section is not in the design and that is why I did not include it. do I still need to add it? thanks.